src/components/RightPane/RightPaneStore.ts

```diff
@@ -54,6 +66,30 @@ class RightPaneStore extends EventEmitter {
         return this.openSpotAlertPopoverActive;
     };
 
+    getUrlCourseCodeValue = () => {
+        return this.urlCourseCodeValue;
+    };
+
+    getUrlTermValue = () => {
+        return this.urlTermValue;
+    };
+
+    getUrlGEValue = () => {
+        return this.urlGEValue;
+    };
+
+    getUrlCourseNumValue = () => {
+        return this.urlCourseNumValue;
+    };
+
+    getUrlDeptLabel = () => {
+        return this.urlDeptLabel;
+    };
+
+    getUrlDeptValue = () => {
+        return this.urlDeptValue;
+    };
```

Comment:
I'm not sure if these would be worth it in the long run; it makes sense to support uni-directional data flows, but it seems like a lot of boilerplate. 

To compromise, here's a re-formatted version that takes up less space; I think it makes sense in this scenario because these are just boilerplate getter methods with no internal logic. 
```suggestion
    getUrlCourseCodeValue = () => this.urlCourseCodeValue;
    getUrlTermValue = () => this.urlTermValue;
    getUrlGEValue = () => this.urlGEValue;
    getUrlCourseNumValue = () =>this.urlCourseNumValue;
    getUrlDeptLabel = () => this.urlDeptLabel;
    getUrlDeptValue = () => this.urlDeptValue;
```